lib/pages/backdrop.dart

```diff
@@ -208,44 +111,89 @@ class _BackdropState extends State<Backdrop>
+      end: RelativeRect.fromLTRB(0, 0, 0, 0),
+    ).animate(
+      CurvedAnimation(
+        parent: _settingsPanelController,
+        curve: Interval(
+          0.0,
+          0.4,
+          curve: Curves.ease,
+        ),
+      ),
+    );
+  }
+
+  Animation<RelativeRect> _slideDownHomePageAnimation(
+      BoxConstraints constraints) {
+    return RelativeRectTween(
+      begin: RelativeRect.fromLTRB(0, 0, 0, 0),
+      end: RelativeRect.fromLTRB(
+        0,
+        constraints.biggest.height - galleryHeaderHeight,
+        0,
+        -galleryHeaderHeight,
+      ),
+    ).animate(
+      CurvedAnimation(
+        parent: _settingsPanelController,
+        curve: Interval(
+          0.0,
+          0.4,
+          curve: Curves.ease,
+        ),
+      ),
+    );
+  }
+
   Widget _buildStack(BuildContext context, BoxConstraints constraints) {
     final isDesktop = isDisplayDesktop(context);
     final safeAreaTopPadding = MediaQuery.of(context).padding.top;
 
-    final Widget frontLayer = ExcludeSemantics(
+    final Widget settingsPage = ExcludeSemantics(
       child: FocusTraversalGroup(
         policy: WidgetOrderTraversalPolicy(),
         child: Focus(
-          skipTraversal: !widget.isSettingsOpenNotifier.value,
-          child: widget.frontLayer,
+          skipTraversal: !_isSettingsOpenNotifier.value,
+          child: SettingsPage(animationController: _settingsPanelController),
         ),
       ),
-      excluding: !widget.isSettingsOpenNotifier.value,
+      excluding: !_isSettingsOpenNotifier.value,
     );
-    final Widget backLayer = ExcludeSemantics(
-      child: widget.backLayer,
-      excluding: widget.isSettingsOpenNotifier.value,
+    final Widget homePage = ExcludeSemantics(
+      child: HomePage(),
+      excluding: _isSettingsOpenNotifier.value,
     );
 
     return FocusTraversalGroup(
       child: InheritedBackdropFocusNodes(
-        frontLayerFocusNode: frontLayerFocusNode,
-        backLayerFocusNode: backLayerFocusNode,
+        settingsPageFocusNode: _settingsPageFocusNode,
+        homePageFocusNode: _homePageFocusNode,
         child: Container(
           child: Stack(
             children: [
+              _galleryHeader(),
               if (!isDesktop) ...[
-                _galleryHeader(),
-                frontLayer,
+                // Slides the settings page up and down from the top of the
+                // screen.
                 PositionedTransition(
-                  rect: _getPanelAnimation(constraints),
-                  child: backLayer,
+                  rect: _slideDownSettingsPageAnimation(constraints),
+                  child: settingsPage,
+                ),
+                // Slides the home page up and down below the bottom of the
+                // screen.
+                PositionedTransition(
+                  rect: _slideDownHomePageAnimation(constraints),
+                  child: homePage,
                 ),
               ],
```

Comment:
I tried it out on mobile and  the animation does not look exactly the same as before now, the staggering is changed.

Before:
![animation_before](https://user-images.githubusercontent.com/1770678/77894738-01cad980-7276-11ea-917e-1b8f3cc656bc.gif)

After:
![animation_after](https://user-images.githubusercontent.com/1770678/77894762-0a231480-7276-11ea-84b7-7b5f4a10ab01.gif)